src/frontend/src/components/mainWindow.ts

```diff
@@ -30,6 +30,16 @@ export const mainWindow = ({
   additionalContainerClasses?: string[];
   HTMLwrapperTag?: string;
 }): TemplateResult => {
+  const pageContent = document.querySelector("#pageContent");
+  const wrap =
+    pageContent !== null ? pageContent : document.querySelector("body");
+
+  // make sure the wrapper always has the l-wrap class, since it is used to
+  // position the content and makes sure it has appropriate spacings
+  if (wrap !== null && wrap.classList.contains("l-wrap") === false) {
```

Comment:
this is a bit hacky, why not wrap the window in another element that contains `l-wrap`?

Reply:
I did that because, we are doing this to be able to accommodate the sidebar later (with l-wrap--sidebar). But I guess I will be solving this problem in the PR of the sidebar then